unabridged-works-volume-1/original/08-to-the-parlaiment.adoc

```diff
@@ -204,10 +213,12 @@ and he will at length bring forth an instrument, which his soul can take pleasur
 and which he will make faithful to him in his work.
 O nation of England, wait on the Lord! wait on the Lord, O my soul!
 
+[.signed-section-context-close]
 From Chalfont in Bucks, the 18. of the 3. mon.
 1659.
 
 This from one who is a friend to the Commonwealth, a lover of true Freedom,
```

Comment:
AHHHH!!! Not sure what to do with this. I tried several different things and they all looked funny to me. I'm sure we've had stuff like this before, but I just can't remember how to work it.

Reply:
Hmmm... my gut is to try a [.postscript]. ...I just tried it locally and I think that it looks good too.  Let's change it to:

```ASCIIDOC
[.postscript]
====

This from one who is a friend to the Commonwealth, a lover of true Freedom,
and desires the good of all men.

====
```